package/ios/RNCSliderManager.m

```diff
@@ -87,6 +87,16 @@ static void RNCSendSliderEvent(RNCSlider *sender, BOOL continuous, BOOL isSlidin
 {
   float value = [sender discreteValue:sender.value];
 
+  if (value < sender.lowerLimit) {
+      value = sender.lowerLimit;
+      [sender setValue:value animated:NO];
+  }
+    
+  if (value > sender.upperLimit) {
+      value = sender.upperLimit;
+      [sender setValue:value animated:NO];
+  }
+
```

Comment:
Same as [the change above](https://github.com/callstack/react-native-slider/pull/432/files#diff-483b3787202f118ea009784458ee16d4020aab95f1d89b3994e0e1af07f33580R119) but for the old arch